src/Microsoft.AspNetCore.Mvc.Razor/RazorPage.cs

```diff
@@ -761,6 +761,8 @@ public Task<HtmlString> RenderSectionAsync(string name)
         /// In layout pages, asynchronously renders the content of the section named <paramref name="name"/>.
         /// </summary>
         /// <param name="name">The section to render.</param>
+        /// <param name="required">Indicates if the section is registered using
+        /// the <c>@section</c> in the Razor page.</param>
```

Comment:
`Indicates the <paramref name="name"/> section must be registered (using <c>@section</c>) in the page.` FYI the section definition is optional if this is `false`.